src/panels/config/zwave/ha-config-zwave.js

```diff
@@ -187,21 +188,57 @@ class HaConfigZwave extends LocalizeMixin(PolymerElement) {
 
         <template is="dom-if" if="[[computeIsNodeSelected(selectedNode)]]">
           <!--Node info card-->
-          <zwave-node-information id="zwave-node-information" nodes="[[nodes]]" selected-node="[[selectedNode]]"></zwave-node-information>
+          <zwave-node-information
+            id="zwave-node-information"
+            nodes="[[nodes]]"
+            selected-node="[[selectedNode]]"
+          ></zwave-node-information>
 
           <!--Value card-->
-          <zwave-values hass="[[hass]]" nodes="[[nodes]]" selected-node="[[selectedNode]]" values="[[values]]"></zwave-values>
+          <zwave-values
+            hass="[[hass]]"
+            nodes="[[nodes]]"
+            selected-node="[[selectedNode]]"
+            values="[[values]]"
+          ></zwave-values>
 
           <!--Group card-->
-          <zwave-groups hass="[[hass]]" nodes="[[nodes]]" selected-node="[[selectedNode]]" groups="[[groups]]"></zwave-groups>
+          <zwave-groups
+            hass="[[hass]]"
+            nodes="[[nodes]]"
+            selected-node="[[selectedNode]]"
+            groups="[[groups]]"
+          ></zwave-groups>
 
           <!--Config card-->
-          <zwave-node-config hass="[[hass]]" nodes="[[nodes]]" selected-node="[[selectedNode]]" config="[[config]]"></zwave-node-config>
+          <zwave-node-config
+            hass="[[hass]]"
+            nodes="[[nodes]]"
+            selected-node="[[selectedNode]]"
+            config="[[config]]"
+          ></zwave-node-config>
+
         </template>
 
+        <!--Protection card-->
+        <template is="dom-if" if="{{_protectionNode}}">
+          <zwave-node-protection
+            hass="[[hass]]"
+            nodes="[[nodes]]"
+            selected-node="[[selectedNode]]"
+            _protection="[[_protection]]"
```

Comment:
should be `protection="[[_protection]]"` then

Reply:
So, ok. There seems to be different views on this. You say it's not going to have and underscore, and balloob says it to have underscore.
My opinion is without underscore, because we pass it up to zwave-node-protection card for further processing, thus not only an internal property.

Reply:
Correct - If anproperty is public/ passed in, we don't use the underscore.